action.yml

```diff
@@ -23,19 +23,34 @@ runs:
       run: |
           REPONAME="${{ github.event.repository.name }}"
           PACKAGENAME=${REPONAME%.jl}
-          echo "::set-output name=packagename::$PACKAGENAME"
+          echo "packagename=$PACKAGENAME" >> $GITHUB_OUTPUT
           if [[ '${{ inputs.test_script }}' == '' ]]; then
             TESTSCRIPT="using ${PACKAGENAME}"
           else
             TESTSCRIPT=${{ inputs.test_script }}
           fi
-          echo "::set-output name=testscript::$TESTSCRIPT"
+          echo "testscript=$TESTSCRIPT" >> $GITHUB_OUTPUT
       shell: bash
       
     - name: Install SnoopCompile tools
       run: julia --project -e 'using Pkg; Pkg.add(["SnoopCompileCore", "SnoopCompile"])'
       shell: bash
     - name: Load package on branch
       id: invs
-      run: julia --project -e 'using SnoopCompileCore; invalidations = @snoopr begin ${{ steps.info.outputs.testscript }} end; using SnoopCompile; inv_owned = length(filtermod(${{ steps.info.outputs.packagename }}, invalidation_trees(invalidations))); inv_total = length(uinvalidated(invalidations)); inv_deps = inv_total - inv_owned; @show inv_total, inv_deps; println("::set-output name=total::$(inv_total)");  println("::set-output name=deps::$(inv_deps)")'
-      shell: bash    
+      run: |
+        using SnoopCompileCore
+        invalidations = @snoopr begin ${{ steps.info.outputs.testscript }} end
+        
+        using SnoopCompile
+        inv_owned = length(filtermod(${{ steps.info.outputs.packagename }}, invalidation_trees(invalidations)))
+        inv_total = length(uinvalidated(invalidations))
+        inv_deps = inv_total - inv_owned
+        
+        @show inv_total, inv_deps
+
+        # Set outputs
+        open(ENV["GITHUB_OUTPUT"], "a") do io
+          println(io, "total=$(inv_total)")
+          println(io, "deps=$(inv_deps)")'
```

Comment:
Typo at end of L54?
Found in https://github.com/JuliaArrays/StructArrays.jl/pull/215 which shows
```julia
ERROR: LoadError: MethodError: no method matching adjoint(::Nothing)
Closest candidates are:
  adjoint(::Union{LinearAlgebra.QR, LinearAlgebra.QRCompactWY, LinearAlgebra.QRPivoted}) at /opt/hostedtoolcache/julia/1.8.2/x64/share/julia/stdlib/v1.8/LinearAlgebra/src/qr.jl:517
  adjoint(::Union{LinearAlgebra.Cholesky, LinearAlgebra.CholeskyPivoted}) at /opt/hostedtoolcache/julia/1.8.2/x64/share/julia/stdlib/v1.8/LinearAlgebra/src/cholesky.jl:558
  adjoint(::LinearAlgebra.Hermitian) at /opt/hostedtoolcache/julia/1.8.2/x64/share/julia/stdlib/v1.8/LinearAlgebra/src/symmetric.jl:348
  ...
```